src/index_spec.coffee

```diff
@@ -167,3 +167,27 @@ describe 'alphabetize_keys', ->
           errors = coffeelint.lint notAlphabetical[name]
           expect(errors).to.have.lengthOf 2
           expect(errors[0].rule).to.eql 'alphabetize_keys'
+
+  context 'overrides', ->
+    context 'public methods', ->
+      context 'alphabetical to overrides', ->
+        it 'returns no errors', ->
+          str = notAlphabetical['define class with public instance methods']
```

Comment:
I'd prefer you just define your own example string in a beforeEach here. That way this block isn't dependent on anything else.


```coffee
context 'overrides', ->
  beforeEach ->
    @str = '''
      class A
        methodC: ->
        methodB: ->
        methodA: ->
      '''
 
   it '...' ->
     #...
     errors = coffeelint.lint @str, opts
     #...